src/controllers/users.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const usersService = require('../services/users');
+
+const getAll = (req, res) => {
+  const users = usersService.getAll();
+
+  res.send(users);
+};
+
+const getOne = (req, res) => {
+  const { userId } = req.params;
+
+  const foundUser = usersService.getById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.statusCode = 200;
+  res.send(foundUser);
+};
+
+const add = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = usersService.create(name);
+
+  res.statusCode = 201;
+  res.send(newUser);
+};
+
+const remove = (req, res) => {
+  const { userId } = req.params;
+
+  const foundUser = usersService.getById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  usersService.remove(userId);
+  res.sendStatus(204);
+};
+
+const update = (req, res) => {
+  const { userId } = req.params;
+  const { name } = req.body;
+
+  const foundUser = usersService.getById(userId);
+
+  if (!foundUser) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  if (typeof name !== 'string') {
```

Comment:
I think we can check for !name, and if it is true, we can return a bad request (400). However, I'm not sure